vxyowsup/whatsapp.py

```diff
@@ -124,8 +132,11 @@ def __init__(self, transport):
         super(WhatsAppInterface, self).__init__()
         self.transport = transport
 
-    def send_to_human(self, text, to_address):
+    def send_to_human(self, text, to_address, message_id):
+        # message_id is vumi id
         message = TextMessageProtocolEntity(text, to=to_address)
+        self.transport.redis.setex(message.getId(), self.transport.config.ack_timeout, message_id)
```

Comment:
`send_to_human` is called in the `asyncore` event loop, so the `redis.setex` call should happen in the Twisted thread somehow. Of course we need the message id, so solving this might be tricky. Just flagging this for the moment.